src/components/Calculator.js

```diff
@@ -1,42 +1,154 @@
 import React from 'react';
 import classes from './Calculator.module.css';
+// import calculate from './logic/calculate';
+// import operate from './logic/operate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-
-    this.state = {
-    };
+    this.state = {};
   }
 
+  handleClick = (e) => {
+    console.log(e.target.value);
+  };
+
   render() {
+    // const { name } = this.state;
+    // console.log(name);
     return (
       <div className={classes.calc__grid}>
         <div className={`${classes.calc__btn} ${classes.btn__1} ${classes.gray}`}><h1 id="output" className={classes.white}>0</h1></div>
-        <div className={classes.calc__btn}><h1>AC</h1></div>
+        <div className={classes.calc__btn}>
+          <button
+            type="button"
+            value="AC"
+            onClick={this.handleClick}
+          >
+            AC
+          </button>
+        </div>
 
-        <div className={classes.calc__btn}><h1>+/-</h1></div>
+        <div className={classes.calc__btn}>
+          <button type="button" onClick={this.handleClick} value="+/-">+/-</button>
+        </div>
 
-        <div className={classes.calc__btn}><h1>%</h1></div>
-        <div className={`${classes.calc__btn} ${classes.chockolate}`}><h1>%</h1></div>
-        <div className={classes.calc__btn}><h1 id="7">7</h1></div>
-        <div className={classes.calc__btn}><h1 id="8">8</h1></div>
-        <div className={classes.calc__btn}><h1 id="9">9</h1></div>
-        <div className={`${classes.calc__btn} ${classes.chockolate}`}><h1>*</h1></div>
-        <div className={classes.calc__btn}><h1 id="4">4</h1></div>
-        <div className={classes.calc__btn}><h1 id="5">5</h1></div>
-        <div className={classes.calc__btn}><h1 id="6">6</h1></div>
+        <button type="button" onClick={this.handleClick} value="%">%</button>
+        <div className={`${classes.calc__btn} ${classes.chockolate}`}>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            value="%"
+          >
+            %
+          </button>
```

Comment:
- [ ] Good job so far 👍 but currently you have 2 `%` buttons and no division button and as a result, division operations cannot be carried out. Kindly rectify this 💯